Repository/AbstractPhpcrRepository.php

```diff
@@ -125,4 +127,39 @@ public function getVersions($path)
             throw NoVersionFoundException::forPath($path, $e);
         }
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function remove($query, $language = 'glob')
```

Comment:
@ElectricMaxxx I think this is for removing multiple resources? But we are only removing one?